reference/forms/types/percent.rst

```diff
@@ -54,6 +56,20 @@ scale
 By default, the input numbers are rounded. To allow for more decimal places,
 use this option.
 
+symbol
+~~~~~~
+
+**type**: ``boolean`` or ``string`` **default**: ``true``
+
+.. versionadded:: 4.3
+
+    The ``symbol`` option was introduced in Symfony 4.3.
+
+By default, fields are rendered with a percentage sign ``%`` after the input.
+Setting the value to ``false`` will not display the percentage sign.
+Setting the value to a `string` (.i.e. ``‱``), will show the string value instead
```

Comment:
Double quotes for `string` please :)

Minor thing: in Symfony Docs we always (I think it's always) use `e.g.` instead of `.i.e.`

Thanks.

Reply:
Will fix it later and try to create a rule for the e.g. / i.e. thingy 👍🏻